matching/clr.py

```diff
@@ -1,17 +1,21 @@
 from eth_utils import is_hex, to_hex
 
+## TODO implement direct read from contract, using...
+##  - infura api
+##  - web3.py
+
 '''
-    Helper function that enforces that funder addresses are in hex format.
+    Helper function that enforces that backer addresses are in hex format.
 
     Args:
-        [funders] (str)
+        [backers] (str)
 
     Returns:
-        [funders] (str)
+        [backers] (str)
 '''
-def check_addresses(funders):
+def check_addresses(backers):
     result = []
-    for f in funders:
+    for f in backers:
```

Comment:
I prefer names like `backer` over single letter variable names (`f` here because it used to stand for funder)

Reply:
me too. will update